.github/workflows/run_tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         shell: bash
     strategy:
       matrix:
-        version: [1.23, 1.24]
+        version: [1.24]
```

Comment:
If this is the only version left, we could also read the Go version from go.mod like we do here:

https://github.com/marcboeker/go-duckdb/blob/main/.github/workflows/tidy_checks.yml#L16